cardano-node/app/cardano-cli.hs

```diff
@@ -461,6 +469,19 @@ parseFlag def active optname desc =
   flag def active $ long optname <> help desc
 
 
+parseTargetNodeAddress :: String -> String -> Parser NodeAddress
```

Comment:
This could be made arguably simpler by breaking it up into something like the following:

`NodeAddress <$> parseHostAddress <*> parsePort`